dist/tools/pyterm/pyterm

```diff
@@ -240,13 +240,25 @@ class SerCmd(cmd.Cmd):
                 sys.exit(1)
         # otherwise go for the serial port
         elif self.port:
+            unconnected = True
```

Comment:
Maybe call this variable `connected` and use `while not connected:` below. This will be more readable.

Reply:
Funny, I had it originally like this, but thought this way it would be more readable ;-).

Reply:
So which way do we go?

Reply:
Using `connected` is more natural way of thinking I would say ;)
```python
connected = False
while not connected:
    try:
        connect()
    except failed:
        exit
    except not ready:
        continue
    else:  # success
        connected = True
```

By the way, `unconnected` doesn't mean 'not connected' but rather 'not related'. The right word would be disconnected I think.

But well, given @cladmi's comment regarding `make test`, I'm wondering if this PR is still worth.

Reply:
Ok... will adapt.